spec/classes/puppet_agent_spec.rb

```diff
@@ -109,7 +109,7 @@ def global_facts(facts, os)
           end
         end
 
-        [{}, {:service_names => []}].each do |params|
+        [{}, {}].each do |params|
```

Comment:
Why run this twice now? This was also meant to test with services managed and unmanaged.